metabase/metabase
# Add stylelint

### Description

Introduce stylelint for css files.

if you think you already saw this PR, you're right https://github.com/metabase/metabase/pull/33033

### How to verify

- add duplicated line to the css file
- try to commit it
- lint-staged should fail
